aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -394,6 +394,15 @@ impl AptosVM {
         )?)
     }
 
+    fn get_senders(txn_data: &TransactionMetadata) -> Vec<AccountAddress> {
+        let mut res = vec![txn_data.sender];
+        res.extend(txn_data.secondary_signers());
+        if txn_data.sequence_number & (1u64 << 63) != 0 {
```

Comment:
What is this doing?

Reply:
Checking if it's a gas payer tx

Reply:
Why do it this way vs a boolean?

Reply:
Adding a boolean would require a whole new format